app/controllers/branding_groups_controller.rb

```diff
@@ -36,16 +36,27 @@ def edit
   # POST /branding_groups.json
   def create
     clean_params = branding_group_params.to_h
-    clean_params[:users] = self.class.merge_curator_params(params[:curator_emails], nil, current_user)
-    clean_params.delete(:user_ids)
+    users = self.class.find_users_from_emails(params[:curator_emails], nil, current_user)
+    clean_params[:user_ids] = users.map(&:id)
+    studies = self.class.find_studies_from_accessions(params[:study_accessions], current_user)
+    clean_params[:study_ids] = studies.map(&:id)
+    missing_studies = self.class.get_missing_studies(
+      self.class.param_to_array(params[:study_accessions]), studies
+    )
     @branding_group = BrandingGroup.new(clean_params)
 
     respond_to do |format|
       if @branding_group.save
+        notice = "Collection '#{@branding_group.name}' was successfully updated."
```

Comment:
Non-blocking: we conventionally use double quotes in user-facing text and put the variable part last.  

Putting variable parts last makes a text message more robust for visual layout in the app and debugging contexts like Mixpanel, Sentry or grep; and makes it easier to search in those debugging contexts.  I don't recall off-hand if these messages get logged via our modal logging to e.g. Mixpanel, but it seems like good practice nonetheless.

```suggestion
        notice = "Successfully updated collection \"#{@branding_group.name}\""
```